### PR TITLE
Improve Exercise groups, feedback, …

### LINKED ISSUE #200: Feedback: Exercise groups
From wolfgang und kathi:

- [x] Über Aufgabengruppen steht in blau "Aufgabengruppe", das sollte weg.
- [x] Uns beiden gefällt der hellblaue strich links nicht
- [x] Bei Aufgabengruppen würden wir die horizontaelen Striche wegmachen und statt dessen ganz - traditionell mit a,b,c, nummerieren. Das aktuell schaut schon iwie fancy aus, aber wir finden es nicht übersichtlich
- [x] Um die Nummerierung 1,2,3 der Aufgaben fänden wir einen Kreis schick
- [x] Wenn man über das cc-Icon hovert, ist der Kreis nicht zentriert.

---

Das grün bei einer richtigen Antwort hebt sich farblich nicht stark vom blau und dem schwarz ab. Ich habe zwar erkannt, dass es eine andere Farbe ist, aber nicht sofort, dass es grün ist und dass meine Antwort deshalb richtig ist.
Zudem ist hier noch ein Bug: Statt des Feedbacks wird bei SC/MC Aufgaben nur die Ausgewählte Antwort rot oder grün angezeigt. Soll ich hierzu ein Issue erstellen oder reicht die Nachricht hier :-)

### LINKED ISSUE #410: Link to Exercise in topicFolder
Allow users in a topic folder to view an exercise separately.

### PR DESCRIPTION
**Pull request by elbotho**
- mostly for #410 and #200 
- a little bit of misc cleanup

Todo:
@Entkenntnis I just saw that `ExerciseGroup` does provide an alias, I'll try to find a way to use that instead of the id.

### CODE REVIEW

**Comment by Entkenntnis** [comment]
Great PR! Cool that you had time to work on #200.

A small idee: If you have a single exercise, we could maybe omit the numbering (the 1): https://frontend-git-add-link-to-exes.serlo.vercel.app/60763. Otherwise, the number of the exercise is changing from the numbr in the group to 1.

Edit: it may require some work to get this information..., so I will open a follow up issue for this.

**Comment by elbotho** [comment]
@Entkenntnis nice catch, should work now 👍

**Comment by elbotho** [comment]
I removed the link to the grouped exercises (the children) for now, without the task of the group it's confusing for users.
We could obviously think about adding the task in the future…

**Comment by elbotho** [comment]
okay, I'm ready here, merge if you agree :)

File: src/fetcher/query.ts [hunk 1]
```diff
@@ -311,6 +311,8 @@ export const dataQuery = gql`
             }
             ... on ExerciseGroup {
               id
+              alias
```
Comment (elbotho) [inline src/fetcher/query.ts:314]: wow, I screwed up something, this is the way I tried first but it did not work.
thanks for the cleaner solution 👍

**Comment by Entkenntnis** [comment]
There is a small style conflict with logged in tools:

grafik

The tools icon is on the left, and the content of the exercise-group is centered.

**Comment by Entkenntnis** [comment]
Otherwise, this PR is ready for merging :tada:

**Review by Entkenntnis — APPROVED** [review]

**Pull request merged**